### PR TITLE
Fixed: Add community contacts page

### PR DESCRIPTION
**Pull request by ahankinson**
Also redirects from the old mailing list page to the new community contacts page.

### CODE REVIEW

**Comment by musicEnfanthen** [comment]
Added two issues connected to this PR. If you like better to see it adressed in another PR, I can provide it after this one is merged.

**Comment by ahankinson** [comment]
@musicEnfanthen Perhaps we can provide a Slack logo next to the other logos? 

We can provide a link to the pages outside of the community section, but they should still be kept inside of the '/community' URL.

**Comment by ahankinson** [comment]
PS: If you want to merge this, go ahead.

File: _config.yml [hunk 1]
```diff
@@ -26,9 +26,9 @@ menu:
           - id: news
             label: 'News / Events'
             url: '/news-events.html'
-          - id: mailinglist
-            label: 'Mailing List'
-            url: '/mailing-list.html'
+          - id: communitycontact
```
Comment (musicEnfanthen) [inline _config.yml]: Could you rename the id "communitycontact[s]" for consistency with label and url?

File: community/community-contacts.md [hunk 2]
```diff
@@ -10,4 +13,11 @@ The list is open to anyone for subscription, and everyone is welcome to join MEI
 
 Also, the archives of the list are open to the public and can be browsed online: [https://lists.uni-paderborn.de/pipermail/mei-l/](https://lists.uni-paderborn.de/pipermail/mei-l/).
 
-Normally, there is not much traffic on the list, but the tone is friendly and most questions are answered within the first 24 hours. So if you have a particular question about MEI, please let others learn from your responses as well by asking there in public.
\ No newline at end of file
+# Slack channel
+
+MEI has a Slack team that is used for discussion, questions, announcements, and interest groups.
```
Comment (musicEnfanthen) [inline community/community-contacts.md]: There isn't anywhere mentioned the name of or a link to the  actual Slack channel. Is this by purpose?  (Would be good for users who do not use it very frequently.)

**Comment by ahankinson** [comment]
Fixed now.

**Comment by musicEnfanthen** [comment]
Thanks, looks great.

Will merge now.

**Pull request merged**